src/group_theory/perm/sign.lean

```diff
@@ -760,4 +883,65 @@ end congr
 
 end sign
 
+section disjoint
+variables [fintype α] {f g : perm α}
+
+lemma disjoint.disjoint_support (h : disjoint f g) :
+  _root_.disjoint f.support g.support :=
```

Comment:
I don't think this dot notation will work here, as the lemma is `equiv.perm.disjoint.disjoint_support`. I'd recomment either calling this `disjoint.disjoint_perm_support`, or `equiv.perm.disjoint_support`.